_posts/2016-01-29-CocoaPods-1.0.markdown

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title:  "CocoaPods 1.0 Migration Guide"
+author: orta
+categories: cocoapods release
+---
+
+#### Major 1.0 changes TL:DR
+​
+
+- You now have to define targets, previously there was an available implicit target. Basically all `pod`s have to be inside a `target`.
+- A `target` has to represent an Xcode target.
+- When you want to represent a collection of pods that go to multiple targets, use an `abstract_target` then add targets inside that. Pod dependencies are inherited by nested targets.
+- Targets that want to know about the pods for a target, but don't need linking (for example, Test targets) can define that they inherit pods via their search paths (`inherit! :search_paths`).
+- `:exclusive => true` and `link_with` have been removed in favour of the above.
+- `xcodeproj` is renamed `project` to make the DSL vocabulary consistent.
+- We've removed the ability to declare `:head` on a `pod` -- you can either use the `:podspec` option or point directly to the upstream repo (via `git`, `:svn`, etc.)
+- You can now declare installation options inside the `Podfile` - via `install!`. (i.e. `install! 'cocoapods', :deterministic_uuids => false`)
```

Comment:
We should mention that `—no-integrate` is such an option now `integrate_targets => false`